### ISSUE TITLE
Exception on invalid qtype 65

### ISSUE THREAD

**Issue by umglurf** [dmachard/pdns-protobuf-receiver#1]
We get a lot of these exceptions. pdns version is 4.3.5

```
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]: 2020-10-14 13:26:46,298 Task exception was never retrieved
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]: future: <Task finished name='Task-15446' coro=<cb_onpayload() done, defined at /usr/local/lib/python3.8/dist-packages/pdns_logger/receiver.py:35
> exception=DNSError('QTYPE: Invalid forward lookup: [65]')>
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]: Traceback (most recent call last):
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]:   File "/usr/local/lib/python3.8/dist-packages/dnslib/bimap.py", line 66, in __getitem__
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]:     return self.forward[k]
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]: KeyError: 65
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]: During handling of the above exception, another exception occurred:
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]: Traceback (most recent call last):
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]:   File "/usr/local/lib/python3.8/dist-packages/pdns_logger/receiver.py", line 87, in cb_onpayload
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]:     dns_msg["query_type"] = dnslib.QTYPE[dns_pb2.question.qType]
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]:   File "/usr/local/lib/python3.8/dist-packages/dnslib/bimap.py", line 68, in __getitem__
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]:     raise self.error("%s: Invalid forward lookup: [%s]" % (self.name,k))
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]: dnslib.dns.DNSError: QTYPE: Invalid forward lookup: [65]
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]: 2020-10-14 13:26:46,300 Task exception was never retrieved
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]: future: <Task finished name='Task-15451' coro=<cb_onpayload() done, defined at /usr/local/lib/python3.8/dist-packages/pdns_logger/receiver.py:35
> exception=DNSError('QTYPE: Invalid forward lookup: [65]')>
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]: Traceback (most recent call last):
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]:   File "/usr/local/lib/python3.8/dist-packages/dnslib/bimap.py", line 66, in __getitem__
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]:     return self.forward[k]
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]: KeyError: 65
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]: During handling of the above exception, another exception occurred:
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]: Traceback (most recent call last):
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]:   File "/usr/local/lib/python3.8/dist-packages/pdns_logger/receiver.py", line 87, in cb_onpayload
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]:     dns_msg["query_type"] = dnslib.QTYPE[dns_pb2.question.qType]
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]:   File "/usr/local/lib/python3.8/dist-packages/dnslib/bimap.py", line 68, in __getitem__
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]:     raise self.error("%s: Invalid forward lookup: [%s]" % (self.name,k))
Oct 14 13:26:46 malxdnsfw01 pdns_logger[1220144]: dnslib.dns.DNSError: QTYPE: Invalid forward lookup: [65]
```

**Comment by dmachard** [comment]
fix in release 0.0.2, the external dnslib has been replaced by "dnspython" library

**Comment by umglurf** [comment]
Great, thanks
